src/styles/cards.scss

```diff
@@ -0,0 +1,101 @@
+.card {
+  width: 200px;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  background: #fff;
+  max-height: 408px;
+  transition: transform 300ms ease;
+  font-family: Roboto, sans-serif;
+
+  &:hover {
+    cursor: pointer;
+    transform: scale(1.2);
```

Comment:
The hover effect using `transform: scale(1.2)` can cause layout shifts, which might lead to a jarring user experience. Consider using a more subtle transformation or alternative effects that do not affect the layout significantly.